lib/rucio/daemons/tracer/kronos.py

```diff
@@ -403,17 +403,13 @@ def kronos_dataset(once: bool = False, dataset_queue: Queue = None, sleep_time:
         )
     )
 
-    # once again for the backlog
+    # once again for the any backlog
```

Comment:
Doesn’t seem like an improvement. Maybe remove the ‘the’ or replace ‘any’ with ‘potential’?